app/routes/__auth/login.ts

```diff
@@ -105,6 +106,7 @@ export const loader: LoaderFunction = async ({ request: { headers, url } }) => {
 
     const user = userFromTokenSet(tokenSet)
     const session = await sessionPromise
+    session.set('accessToken', tokenSet.access_token)
     Object.entries(user).forEach(([key, value]) => {
       session.set(key, value)
     })
```

Comment:
This looks like a copy of `updateSession`.

Reply:
Yea, originally the two looked a bit different, but I refactored a bit and I think it is a bit cleaner now. Also added a simple comment on the method to give it some clarity in use.